packages/client.api/src/object/Attachment.ts

```diff
@@ -17,3 +17,19 @@
 export interface Attachment {
   rid: string;
 }
+
+export interface AttachmentUpload extends Blob {
```

Comment:
JSDoc that mentions that the `File` object from the W3C FileApi https://www.w3.org/TR/FileAPI/#file-section satisifies this?